### PR TITLE
moving more extension search routes to /ext/

### PR DESCRIPTION
**Pull request by 2is10**
@leogrim 
cc @ymatsuda

### CODE REVIEW

File: kifi-backend/modules/search/conf/search.routes [hunk 1]
```diff
@@ -6,13 +6,22 @@ GET         /admin/search/instance         @com.keepit.controllers.search.Search
...
+POST        /ext/search/prime              @com.keepit.controllers.ext.ExtSearchController.warmUp()
+
 GET         /search/message        @com.keepit.controllers.ext.ExtMessageSearchController.search(q: String, page: Int ?= 0)
+
+# Note:     /search/events routes are deprecated, were last used in extension 3.2.16
 POST        /search/events/resultClicked        @com.keepit.controllers.ext.ExtSearchEventController.clickedSearchResult()
 POST        /search/events/searched        @com.keepit.controllers.ext.ExtSearchEventController.searched()
-POST        /search/events/browsed        @com.keepit.controllers.ext.ExtSearchEventController.updateBrowsingHistory()
+POST        /ext/search/events/searched    @com.keepit.controllers.ext.ExtSearchEventController.searched()
+POST        /ext/search/events/clicked     @com.keepit.controllers.ext.ExtSearchEventController.clickedSearchResult()
```
Comment (2is10) [inline kifi-backend/modules/search/conf/search.routes:23]: Notice I changed a couple of camel-cased two-word path segments to one-word alternatives. I try to keep our public routes lowercase, simple and RESTful when possible. Our conventions for internal routes, of course, are different.

**Comment by leogrim** [comment]
lgtm

**Comment by 2is10** [comment]
Thanks for reviewing.

In the future, if there’s a merge conflict in one of my pull requests, I’ll be happy to resolve and merge it myself later.

**Comment by leogrim** [comment]
I think there only was because I merged them in the wrong order, so entirely my fault.

**Pull request closed without merging**